### PR TITLE
Contact form updated

### PR DESCRIPTION
**Pull request by Ndaboom**
Section's title color has been updated and fields requirement such as the name has been set up to 30 chars and messages up to 500 chars.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -203,9 +203,9 @@ <h3 class="work_title">Skills</h3>
     <section id="form_section">
         <h1 class="primary-font-style contact_form_text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
         <form class="input_form">
```
Comment (aimalamiri) [inline index.html]: Good job, but please make sure to add the `method` attribute with a proper value and the `action` attribute with the link to the [Formspree](https://formspree.io/html) form.

File: index.html [hunk 2]
```diff
@@ -203,9 +203,9 @@ <h3 class="work_title">Skills</h3>
     <section id="form_section">
         <h1 class="primary-font-style contact_form_text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
```
Comment (aimalamiri) [inline index.html:207]: It looks nice, but it's a requirement to stick with the Figma design, so please try to follow the exact font size and font weight.

image

File: index.html [hunk 3]
```diff
@@ -203,9 +203,9 @@ <h3 class="work_title">Skills</h3>
...
         <form class="input_form">
-            <input type="text" class="form-input classic" name="fullname" placeholder="Full name" required/>
+            <input type="text" class="form-input classic" name="fullname" maxlength="30" placeholder="Full name" required/>
             <input type="email" class="form-input classic" name="email" placeholder="Email address" required/>
-            <textarea class="form-input classic-md" name="message" placeholder="Enter text here" required></textarea>
+            <textarea class="form-input classic-md" name="message" maxlength="500" placeholder="Enter text here" required></textarea>
```
Comment (aimalamiri) [inline index.html]: Your design looks very similar to the one in Figma, but please add some padding to the `inputs` and also decrease the space between inputs as you see in the Figma template.

image

File: index.html [hunk 4]
```diff
@@ -203,9 +203,9 @@ <h3 class="work_title">Skills</h3>
...
             <input type="submit" class="btn-primary primary-font-style btn-connect" name="submit-form" value="Get in touch" />
         </form>
         <ul id="social_media_links">
```
Comment (aimalamiri) [inline index.html]: Good job! but I'd recommend fixing the alignment and sizing issue for the social media icons, to make them more beautiful.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @Ndaboom, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Ndaboom) [reply index.html:207]: Hi @aimalamiri, the font-weight CSS property is not working as expected. Still looking for a way out.

File: index.html [hunk 5]
```diff
@@ -202,10 +202,10 @@ <h3 class="work_title">Skills</h3>
...
     <section id="form_section">
         <h1 class="primary-font-style contact_form_text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
-        <form class="input_form">
-            <input type="text" class="form-input classic" name="fullname" placeholder="Full name" required/>
+        <form class="input_form" action="https://formspree.io/f/maykozkk" method="POST">
+            <input type="text" class="form-input classic" name="fullname" maxlength="30" placeholder="Full name" required/>
             <input type="email" class="form-input classic" name="email" placeholder="Email address" required/>
-            <textarea class="form-input classic-md" name="message" placeholder="Enter text here" required></textarea>
+            <textarea class="form-input classic-md" name="message" maxlength="500" placeholder="Enter text here" required></textarea>
             <input type="submit" class="btn-primary primary-font-style btn-connect" name="submit-form" value="Get in touch" />
```
Comment (roshan-bajgain) [inline index.html:212]: - [ ] As requested by the previous reviewer please make sure to stick to the Figma design. There are 5 social icons and please fix the spacing and gaping as well to make similar to the design
### your design
his
### Figma design
figma

File: index.html [hunk 6]
```diff
@@ -202,10 +202,10 @@ <h3 class="work_title">Skills</h3>
     <!-- Contact form section -->
     <section id="form_section">
         <h1 class="primary-font-style contact_form_text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
```
Comment (roshan-bajgain) [inline index.html:207]: - [ ] As requested by the previous reviewer please make sure to use the `font-family: 'Inter';` to stick to the design
### Your design
his
### Figma design
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Ndaboom, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Ndaboom 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Great job making the style as close as much as the Figma design. ✅ ✅ ✅ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). Please remember this in future projects.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**